infra/tofu_importable_modules/bloom_deployment/ecs_site_public_task.tf

```diff
@@ -0,0 +1,165 @@
+resource "aws_ecs_task_definition" "bloom_site_public" {
+  region                   = var.aws_region
+  family                   = "bloom-site-public"
+  network_mode             = "awsvpc"
+  requires_compatibilities = ["FARGATE"]
+  runtime_platform {
+    operating_system_family = "LINUX"
+    cpu_architecture        = "X86_64"
+  }
+
+  execution_role_arn = aws_iam_role.bloom_ecs["site-public"].arn
+  task_role_arn      = aws_iam_role.bloom_container["site-public"].arn
+
+  # https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definition_parameters.html#task_size
+  cpu    = 2048     # 2 vCPU
+  memory = 6 * 1024 # 4 GiB in MiB
+
+  container_definitions = jsonencode([
+    {
+      Name  = "bloom-site-public"
+      image = var.bloom_site_public_image
+      environment = [
+        {
+          name  = "NODE_ENV"
+          value = "production"
+        },
+        {
+          name  = "NEXTJS_PORT"
+          value = "3000"
+        },
+        {
+          name  = "BACKEND_API_BASE"
+          value = "http://bloom-api:3100"
+        },
+        {
+          name  = "BACKEND_API_BASE_NEW"
+          value = "http://bloom-api:3100"
+        },
+        {
+          name  = "LISTINGS_QUERY"
+          value = "/listings"
+        },
+        {
+          name  = "MAX_BROWSE_LISTINGS"
+          value = "10"
+        },
+        {
+          name  = "HOUSING_COUNSELOR_SERVICE_URL"
+          value = "/get-assistance"
+        },
+        {
+          name  = "JURISDICTION_NAME"
+          value = "Bloomington"
```

Comment:
question: I know this is just for the first pass, is there an issue for how to make these environment variables configurable? Many of these will be the same for all deployments, but things like jurisdiction name and languages will change.